zh_cn/13.常见问题/01.debug-system-faq.md

```diff
@@ -1,25 +1,9 @@
 ---
-title: 系统故障 FAQ
+title: 部署运维 FAQ
 date: 2021-03-31 09:32:30
```

Comment:
Does the date need to be updated?

Reply:
It is just some information of when the file was created(generated automaticly). used nowhere for now.